### PR TITLE
Time offset

### PR DESCRIPTION
**Pull request by magland**

### CODE REVIEW

**Comment by magland** [comment]
@jsoules you are welcome to review this. I am going to merge now b/c there is very low chance of breaking something.

Added PositionPlot class which should be used instead of the old create_position_plot(). This conforms with other of the recent style views, and allows including in a layout sync'd with other views.

Added deprecated message to create_position_plot()

Here's the real change: Allow float64 timestamps by passing in timeOffset and float32 timestamps that are referenced to the first timestamp. The corresponding updates have been added to spikesortingview.

**Comment by magland** [comment]
For the record, I posted this on slack:

Update on position plot and float64 timestamps.
* Upgrade to latest sortingview (>=0.9.11)
* You can now pass in float64 timestamps to position plot
create_position_plot() has been deprecated, and you will get a helpful warning if you use it (but it should work still)
* Use vv.PositionPlot(...) instead, because this will allow you to compose it with other views in the newer layout system - for example you can put a position plot side-by-side with a track animation view
* Re float64 timestamps: The way this works internally is that the first timestamp is automatically passed to browser as timeOffset, and the array of timestamps is automatically referenced to this timestamp and the cast to float32 for the data transfer
* In other words, we still don't support float64 for other fields -- timestamps is treated specially as above, and transferred as float32 after referencing.
